API-strategie-extensies/ext-geospatial.md

```diff
@@ -117,12 +117,12 @@ The default CRS (Coordinate Reference System) for GeoJSON is WGS84. This is the
 
 Since most client-side mapping libraries use WGS84, the W3C/OGC working group *Spatial Data on the Web* recommends to use this as the default coordinate reference system. Thus, spatial data can be mapped without any complex transformations. The API strategy caters for this supporting not only ETRS89 and RD/Amersfoort, but also WGS84 and Web Mercator (EPSG:3857).
 
-<div class="rule" id="api-39">
-  <p class="rulelab"><strong>API-39</strong>: Use ETRS89 as the preferred coordinate reference system (CRS)</p>
+<div class="rule" id="api-40">
+  <p class="rulelab"><strong>API-40</strong>: Use ETRS89 as the preferred coordinate reference system (CRS)</p>
   <p>General usage of the European ETRS89 coordinate reference system (CRS) is preferable, but is not necessarily the default CRS. Hence, the CRS has to be explicitly included in each request.</p>
 </div>
 
-The CRS can be specified for request and response individually using parameters: RD/Amersfoort, ETRS89, WGS84, and Web Mercator.
+The CRS can be specified for request and response individually using parameters and headers.
```

Comment:
The CRS can be specified for request and response individually using parameters or headers.